cmd/ndm_daemonset/controller/blockdevice.go

```diff
@@ -17,6 +17,7 @@ limitations under the License.
 package controller
```

Comment:
Can this file be completely replaced by `blockdevice/blockdevice.go` ?

Reply:
Not at the moment @kmova . We are generating the `BlockDevice` resource from this struct. Since we haven't written any methods for our new core blockdevice struct to convert to `BlockDevice` resource, we cannot remove this file as of now. 

Once all the client methods are implemented for the new core struct, we can remove this. Those changes will be taken care as part of another PR.